lib/verbalize/action.rb

```diff
@@ -19,46 +16,24 @@ def self.included(target)
     end
 
     module ClassMethods
-      def input( # rubocop:disable Metrics/MethodLength
-        *required_keywords,
-        optional: [],
-        **other_keyword_arguments
-      )
-        unless other_keyword_arguments.empty?
-          raise(
-            ArgumentError,
-            "Unsupported keyword arguments received: #{other_keyword_arguments.inspect}"
-          )
+      def input(*required_keywords, optional: [], **unsupported_keywords)
```

Comment:
Also, we should raise here if both arrays are empty, since it will break method creation in `Build` if you call `input` with no parameters